test/unittest/unittest_nntrainer_layers.cpp

```diff
@@ -1911,6 +1915,7 @@ TEST(nntrainer_LossLayer, forward_loss_unknown_n) {
     layer.getType(), layer.getName(), layer.getOutputDimension()));
 
   manager.initializeTensors(true);
+  manager.allocateTensors();
```

Comment:
We need this from now on.

Reply:
yes, when using without model.
model.inference() and model.train() call this internally.